<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitializationWindow</class>
 <widget class="QMainWindow" name="InitializationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>404</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,4,1">
    <item row="0" column="0">
     <widget class="QLabel" name="lblLogo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources.qrc">:/Logo2.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ls_init_status">
      <property name="styleSheet">
       <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>INITIALIZATION STATUS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="ls_init_switch">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Initialization Switch</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_init_switch">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_system_status">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ls_handpiece">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Handpiece Connection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ls_system_status">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>System Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_handpiece">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_pedal_con">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ls_pedal_con">
        <property name="styleSheet">
         <string notr="true">color : rgb(255,255,255);
selection-background-color:rgb(238,236,237);
selection-color:rgb(238,236,237);
alternate-background-color:rgb(0,0,0);
font:bold 75 16pt &quot;Noto Sans&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>PEDAL Connection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="btn_reinit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font:bold 75 16pt &quot;Noto Sans&quot;;
color: #333;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}</string>
      </property>
      <property name="text">
       <string>Retry Initialization</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
